### PR TITLE
KOGITO-4222 Added some cloud methods for Quay

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-4222

Related PRs:
- https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/134
- https://github.com/kiegroup/kogito-images/pull/327

### CODE REVIEW

**Comment by radtriste** [comment]
Tested here: https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/tradisso/job/quay%20api/25/console
made this repository public => https://quay.io/repository/tradisso/kogito-test-cloud-operator

**Review by Ginxo — APPROVED** [review]
LGTM, just check my comment

File: vars/cloud.groovy [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+* Make a quay.io image public if not already
+*/ 
+void makeQuayImagePublic(String namespace, String repository, String quayTokenCredentialsId) {
+    if(!isQuayImagePublic(namespace, repository, quayTokenCredentialsId)){
```
[the remark this answers is not in the document]
Comment (radtriste) [reply vars/cloud.groovy:5]: well that is the same but indeed, we could merge in one `if`

**Comment by radtriste** [comment]
@Ginxo update with your comment. Feel free to merge once checks are ok :)

**Pull request merged**